openerp/tools/config.py

```diff
@@ -443,6 +443,7 @@ def die(cond, msg):
             'test_file', 'test_enable', 'test_commit', 'test_report_directory',
             'osv_memory_count_limit', 'osv_memory_age_limit', 'max_cron_threads', 'unaccent',
             'data_dir',
+            'server_wide_modules',
```

Comment:
Hi @StefanRijnhart,

Yes, it would be more cosmetic to add it to the same line, but it's exactly the risk of conflicts that made me add a new line. There is much less conflict risk in of conflict in a line addition than a line change (as would be required to keep it cosmetic).

In any case if you insist, I can add the option in the same line, but that will make it diverge from the same PR in odoo proper.